src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -844,6 +852,45 @@ public void processPresence(Presence presence)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public boolean isAvModerationEnabled()
+    {
+        return avModerationEnabled;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void setAvModerationEnabled(boolean value)
+    {
+        this.avModerationEnabled = value;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public void updateAvModerationWhitelists(@NotNull Map<String, List<String>> whitelists)
+    {
+        this.whitelists = whitelists;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public boolean isMemberAllowedToUnmute(Jid jid, MediaType mediaType)
+    {
+        if (!this.isAvModerationEnabled())
+        {
+            return true;
+        }
+
+        List<String> whitelist = this.whitelists.get(mediaType.toString());
```

Comment:
Can we make the `whitelist` field `@NotNull` (it's already effectively not null) instead of doing null checks?

Reply:
This is what is received from the client and the list indeed can be null.

Reply:
Ah, of course, I misread.